news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,178 @@
+- **You must have been a member of the Beatmap Nominators in the past year**. Additionally you must have left on "good terms". On "good terms" means that you have passed your last evaluation. If both apply to you, you will only need **one month** of modding activity in order to apply!
+
+This has been tested on leaving Beatmap Nominators for a few months already, so you may already have this leniency if you've left recently.
+We've got your interest? Great! When you are sure that the activity lenience can be applied to you, contact any member of the Nomination Assessment Team! (Preferably of your game mode, because they will most likely know if you were a Beatmap Nominator in the past.)
+
+### BN application and evaluation results will now be sent via chat
+
+Next, the NAT has an important announcement to make regarding future Beatmap Nominator evaluations and applications. As you may know, all results used to be delivered through legacy pm on the old osu! website. **This is no longer the case**. You will now receive a message through the website/in-game chat function once your application or evaluation has concluded. This message will contain a link that will lead you to the **Beatmap Nominator website**, where more detailed feedback can be viewed. 
+
+And now let's go to the team movements for this month:
+
+### BN Additions
+
+- **osu!:** [-Aqua](https://osu.ppy.sh/users/7150015), [Bibbity Bill](https://osu.ppy.sh/users/4446810), [Log Off Now](https://osu.ppy.sh/users/4378277), [C00L](https://osu.ppy.sh/users/4930630), [Ayucchi](https://osu.ppy.sh/users/7823498) & [Hobbes2](https://osu.ppy.sh/users/8157492)
+- **osu!catch:** [wonjae](https://osu.ppy.sh/users/5032045) & [-Joni-](https://osu.ppy.sh/users/9988837)
+- **osu!mania:** [Arzenvald](https://osu.ppy.sh/users/3027421), [lenpai](https://osu.ppy.sh/users/5314573) & [Davvy](https://osu.ppy.sh/users/10047413)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. 
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!:** [Peter](https://osu.ppy.sh/users/8623835), [Kaitjuh](https://osu.ppy.sh/users/2225327), [Icekalt](https://osu.ppy.sh/users/5410645) & [Rizen](https://osu.ppy.sh/users/3056732)
+- **osu!taiko:** [Volta](https://osu.ppy.sh/users/4154071)
+- **osu!catch:** [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+Another month, another Mappers' Showcase! However, "mapper" is maybe not the right term this time. Since we covered the first round of all game modes, we thought we'd make something special this time. We would like to show you the work of storyboarders!
```

Comment:
```suggestion
Another month, another Mappers' Showcase! However, the term "mapper" is maybe not the right term this time. Since the showcase has covered all game modes at least once, we thought we'd make something special this time. We would like to show you the work of storyboarders!
```

Reply:
There's no need to mention "term" twice in the sentence. Seemed fine as it was.

Reply:
good point, changed the second one just so the sentence flows better